### ISSUE TITLE
IPv6 bind failure in github workflow build

### ISSUE THREAD

**Issue by iSignal** [yugabyte/yugabyte-db#4851]
@mbautin pointed out this failure to bind to '[::1]' in the github workflow at https://github.com/yugabyte/yugabyte-db/runs/787021393.

This is likely because yb-master run during initial sys catalog snapshot test is created with the flags `--webserver_interface=localhost` which then resolves to both 127.0.0.1 and [::1] (we now ask for ipv4 and ipv6 results both in getaddrinfo via AF_UNSPEC). For some reason binding to [::1] fails on the workflow host.

One option to fix this is to change the default at https://github.com/yugabyte/yugabyte-db/blob/fe6b22040b27c88b7b2045d6ff982844823afd8f/src/yb/util/net/net_util.cc#L82 to only use IPv4 addreses by default. This would make it equivalent to doing a getaddrinfo via AF_INET as was done previously.

I will attempt to reproduce this on other clean hosts to understand (a) why the bind to '[::1]' fails and (b) how widespread the problem is.

cc: @spolitov @bmatican

**Comment by mbautin** [comment]
For localhost, do we really need IPv6? Can we implement a special case for resolving localhost?

**Comment by iSignal** [comment]
@mbautin : The issue isn't resolution. On the build machine, localhost resolves to both 127.0.0.1 and [::1] correctly like on hosts that support both stacks. However binding to [::1] fails.

```
2020-06-19T07:00:22.1922313Z [m-1] I0619 07:00:21.438045 33003 webserver.cc:150] Webserver listen spec is 127.0.0.1:26926,[::1]:26926
2020-06-19T07:00:22.1923607Z [m-1] I0619 07:00:21.438467 33003 webserver.cc:309] Webserver: set_ports_option: cannot bind to [::1]:26926: 99 (Cannot assign requested address)
```
